packages/terra-data-grid/tests/wdio/flowsheet-data-grid-spec.js

```diff
@@ -1,13 +1,140 @@
 const defaultSelector = '#default-terra-flowsheet-data-grid';
 
+const holdDownShiftKey = () => {
+  browser.performActions([{
+    type: 'key',
+    id: 'keyboard',
+    actions: [{ type: 'keyDown', value: '\uE008' }],
+  }]);
+};
+
+const releaseShiftKey = () => {
+  browser.performActions([{
+    type: 'key',
+    id: 'keyboard',
+    actions: [{ type: 'keyUp', value: '\uE008' }],
+  }]);
+};
+
+const moveCurrentPositionBy = (row, col) => browser.keys(
+  new Array(Math.abs(row)).fill(row > 0 ? 'ArrowDown' : 'ArrowUp')
+    .concat(new Array(Math.abs(col)).fill(col > 0 ? 'ArrowRight' : 'ArrowLeft')),
+);
+
+const navigateToCell = (row, col) => {
+  browser.keys(['Tab']);
+  moveCurrentPositionBy(row, col);
+};
+
+const clickCell = (row, col, selector) => {
+  browser.$$(`${selector} tr`)[row].$(`:nth-child(${col + 1})`).click();
+};
+
 Terra.describeViewports('FlowsheetDataGrid', ['medium', 'large'], () => {
-  describe('Default Flowsheet Data Grid', () => {
+  describe('Cell selection', () => {
     beforeEach(() => {
       browser.url('/raw/tests/cerner-terra-framework-docs/data-grid/flowsheet-data-grid/default-flowsheet-data-grid');
     });
 
-    it('should display the default flowsheet data grid', () => {
-      Terra.validates.element('default', { selector: defaultSelector });
+    it('selects the current cell with spacebar', () => {
+      navigateToCell(1, 1);
+      browser.keys(['Space']);
+
+      Terra.validates.element('selection-and-focus-on-1-1', { selector: defaultSelector });
```

Comment:
can we add focus checks to these tests?

Reply:
Added

Reply:
Let's make the expect first so that the if the screenshots fail due to a pixel shift, we can still verify that the behavior is correct